### DISCUSSION TITLE
Unable to bring the sensor up with an Goouuu Mini-S1

### DISCUSSION THREAD

**Discussion by sorryusernameisalreadytaken** [TheRealWaldo/esp8266-amg8833#8] [Q&A]
As far as I understand the code now, it always uses the standard I2C pins of the respective board (https://github.com/platformio/platform-espressif8266/tree/master/boards) which you can specify in the platformio.ini. Am I right **?**

Unfortunately I can't get my amg8833 to work. At least I guess the ESP8266 website would give me some data at the end instead of "Not found", right **?**

I have unfortunately only two ESP8266 with the label **"Goouuu Mini-S1"** available. Here you can find a few info:
https://www.esp8266learning.com/goouuu-s1-esp8266.php
http://twincati.blogspot.com/2017/02/goouuu-s1-esp8266.html

And here are some github results too:
https://github.com/Bob0505/Goouuu_Mini-S1
https://github.com/smarthomefans/esphome-template/blob/dacd9ea1f13d71cc18e31b60fa3871a46c6e7359/wifi_state_led/wifi_state_led.yaml
https://github.com/Siorinex/ESPHome-samples/commit/df9837f6f085dd9ce49c1eae7b635f44e6c93e90
https://github.com/Siorinex/ESPHome-samples/commit/993373cf86ea54bd5ba394d89e00070a89992d8e

The web-ui gives me this:
![grafik](https://user-images.githubusercontent.com/12913436/228033838-8038f4a1-bec4-41dd-be7f-7b7a9176d97b.png)

And the cmd when I upload my bin tells something too about ESP8266EX.

I do not understand why my board seems to be so tricky to get the amg8833 to work.

**Comment by TheRealWaldo** [comment]
I'm afraid I cannot speak to that board personally. This is the first time I have encountered it.

You definitely need to ensure that the board profile you set in the configuration matches your board.  There are many variants with different configurations!

Also, if your board doesn't support the Wire library, you may need to map the pins explicitly in the code.

You may have better luck asking about which profile to use for this board on the Platform.io boards, to which there is a larger audience!  It might match one that is already defined, or you may need to do a little digging to create and configure a profile specific to your board.

**Reply by sorryusernameisalreadytaken** [reply]
Can you help me a bit with the Wire library? In the past I use this board a lot with mapping the pins explicitly in the code. But I never use the Wire library.

I think the board is a strange clone of a wemos d1 mini or something similar. When I translate this mandarin CN string here:
https://github.com/smarthomefans/esphome-template/blob/dacd9ea1f13d71cc18e31b60fa3871a46c6e7359/wifi_state_led/wifi_state_led.yaml#L3
--> "# board: ESP8266-S1 Since PIO does not support this board, the copycat board only has 2M Flash so choose esp8285"

Yesterday I try a lot of different pins and board types. Without any luck :(

**Reply by TheRealWaldo** [reply]
Check out https://www.arduino.cc/reference/en/language/functions/communication/wire/

If you're using this for Home Assistant, you should consider using my ESPHome variant, which you may have better luck with.
